package.json

```diff
@@ -633,46 +627,23 @@
             "description": "Editor server default port",
             "default": 9000
           },
-          "dataEditorLogFile": {
-            "type": "string",
-            "description": "Path to log file for data editor",
-            "default": "${workspaceFolder}/dataEditor-${omegaEditPort}.log"
-          },
-          "dataEditorLogLevel": {
-            "type": "string",
-            "description": "Log level for data editor",
-            "enum": [
-              "error",
-              "warn",
-              "info",
-              "debug",
-              "trace"
-            ],
-            "default": "info"
-          },
-          "dfdlDebuggerLogLevel": {
-            "type": "string",
-            "description": "Log level for debugger",
-            "enum": [
-              "ALL",
-              "DEBUG",
-              "ERROR",
-              "INFO",
-              "OFF",
-              "TRACE",
-              "WARN"
-            ],
-            "default": "INFO"
-          },
-          "dfdlDebuggerLogFile": {
-            "type": "string",
-            "description": "Path to log file for debugger",
-            "default": "/tmp/daffodil-debugger.log"
+          "dataEditor": {
```

Comment:
@scholarsmate I think we'll have to modify the `dataEditorClient.ts` to accommodate this.  

https://github.com/ctc-oss/daffodil-vscode/commit/257ce11c6181dfc097b5d953dc440dfda8323d0d - Looks like we changed the way logging variables were pulled in from the VSCode configuration. I'd assume `logLevel` and `logFile` are similar.

Reply:
Right.  If this gets merged, we'll need to update `dataEditorClient.ts` to match.